.gitignore

```diff
@@ -108,6 +108,7 @@ logs/
 [Rr]elease/
 x64/
 build/
+build*/
```

Comment:
This adds `modules/mono/build_scripts/` to the ignore list. I'm going to rename it to `modules/mono/meson_scripts`, but adding gitignore entries like is to ask for trouble IMO.

Reply:
If you start this with a slash, Git will only ignore it in the top source directory. So `/build*` would match a top level `build-linux` but not `modules/mono/build_scripts´.

http://git-scm.com/docs/gitignore